spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -235,4 +235,13 @@
       expect(current_path).to eq profile_path
     end
   end
+
+  describe 'clicking the logo image during 2fa process' do
+    it 'signs the user out and returns them to the home page' do
+      user = build_stubbed(:user, :signed_up)
+      sign_in_user(user)
+      find("img[alt='login.gov']").click
+      expect(current_path).to eq root_path
```

Comment:
Perhaps here we should check for something else that is more directly related to being logged out? (I assume that logged in users can visit `root_path` ...)